.travis.yml

```diff
@@ -55,17 +53,13 @@ script:
   - if [ -n "$STATIC_CHECKERS" ]; then ./bin/test-without-stubs fmt; fi
   - if [ -n "$STATIC_CHECKERS" ]; then ./bin/test-without-stubs vet; fi
   - if [ -z "$STATIC_CHECKERS" ]; then ./bin/test-without-stubs; fi
-  - if [ -n "$STATIC_CHECKERS" ]; then
-        ./bin/test-without-stubs lint '--disable=errcheck,megacheck --enable=goimports,goconst,gocyclo';
-    fi
+  - if [ -n "$STATIC_CHECKERS" ]; then ./bin/test-without-stubs lint '--disable=errcheck,megacheck --enable=goimports,goconst,gocyclo'; fi
 
```

Comment:
just a preference for one line if I suppose.  i don't see a content change.  I have to scroll to the right to read it, but I'm ok with it.

Reply:
multi-line requires continuation character and open to errors, so I guess one line is better. 

I formatted automatically.

Reply:
ok!